workflow/daymet.py

```diff
@@ -57,7 +57,9 @@ def initData(d, vars, num_days, nx, ny):
         # d[v] has shape (nband, nrow, ncol)
         d[v] = np.zeros((num_days, ny, nx),'d')
 
-def collectDaymet(bounds, crs, start, end, vars=None, force=False, combine=False):
+
+def collectDaymet(bounds, start, end, crs, vars='all', force=False, combine = False, buffer = 0.01):
```

Comment:
Please follow the style conventions of the rest of the file/project -- no spaces around optional arguments

`combine=False, buffer=0.01`